src/python/pants/backend/jvm/tasks/jvm_compile/rsc/rsc_compile.py

```diff
@@ -181,6 +189,14 @@ def register_options(cls, register):
       ]
     )
 
+  @memoized_property
+  def _rsc_native_image(self):
+    return RscNativeImage.scoped_instance(self)
```

Comment:
I don't think that this should be scoped: should use `RscNativeImage.global_instance()` instead (the same way `Zinc` is consumed).

The reason for this is that there isn't really a good reason to have different versions of rsc in different parts of the codebase anytime soon.